src/Acuminator/Acuminator.Utilities/Roslyn/PXFieldAttributes/AttributeInformation.cs

```diff
@@ -183,11 +195,25 @@ public BoundType GetBoundAttributeType(AttributeData attribute)
 			{
 				var isDbPropertyAttributeArgs = attribute.NamedArguments.Where(arg => IsDBField.Equals(arg.Key, StringComparison.OrdinalIgnoreCase)).ToList();    //case insensitive check
 
+				if (!(isDbPropertyAttributeArgs[0].Value.Value is bool isDbPropertyAttributeArgument))
+					return BoundType.Unknown;  //if there is null or values of type other than bool then we don't know if attribute is bound
+
 				if (isDbPropertyAttributeArgs.Count != 1)  //rare case when there are multiple different "IsDBField" considered
 					return BoundType.Unknown;
 
-				if (!(isDbPropertyAttributeArgs[0].Value.Value is bool isDbPropertyAttributeArgument))
-					return BoundType.Unknown;  //if there is null or values of type other than bool then we don't know if attribute is bound
+				var isDBFieldSetInFalseInBaseAttribute = TypesContainingIsDBField.Any(t => 
+					t.Key != null && t.Value == false && IsAttributeDerivedFromClass(attribute.AttributeClass, t.Key));
+
+				var isDBFieldSetInTrueInBaseAttribute = TypesContainingIsDBField.Any(t =>
+					t.Key != null && t.Value == true && IsAttributeDerivedFromClass(attribute.AttributeClass, t.Key));
```

Comment:
@saloponov 
1) The dictionary can't have a null in its keys so the check `t.Key != null` is redundant
2) You run quite expensive check `IsAttributeDerivedFromClass` twice. You can just write it like this:
```csharp
var typeFromRegister = TypesContainingIsDBField.Keys.FirstOrDefault(t => IsAttributeDerivedFromClass(attribute.AttributeClass, t);
 bool? isDbFieldFromBase = typeFromRegister != null
    ? TypesContainingIsDBField[t]
    : (bool?) null;

if (isDbPropertyAttributeArgs.Count == 0 && isDbFieldFromBase.HasValue)
{	
      return isDbFieldFromBase.Value 
          ? BoundType.DbBound
          : BoundType.Unbound;				
}

//Rest of code
```